pom.xml

```diff
@@ -20,6 +20,7 @@
         <module>iot-e2e-tests</module>
         <module>provisioning</module>
         <module>digital-twin</module>
+        <module>pnp-helper</module>
```

Comment:
I like this name, I had named the module "pnp" in my PR, but I can take this change if your PR goes in first.